### PR TITLE
Pass background color to JS renderer

### PR DESCRIPTION
**Pull request by eulertour**
Passes the background data so it can be used with the JS renderer after https://github.com/ManimCommunity/manim-renderer/commit/c09a5e5a5663ddc618c9f47f74d6d5cc326248aa.

Also sets and updates the `fill_opacity` and `stroke_opacity` attributes for ImageMobjects, so that their opacity will be sent to the JS renderer https://github.com/ManimCommunity/manim-renderer/commit/8e89884ac74f87e6b60e0d4a58c60c161bec9d2b.

### CODE REVIEW

File: manim/mobject/types/image_mobject.py [hunk 1]
```diff
@@ -93,7 +93,8 @@ def __init__(
         image_mode="RGBA",
         **kwargs,
     ):
-        self.opacity = 0
+        self.fill_opacity = 1
```
Comment (leotrs) [inline manim/mobject/types/image_mobject.py:96]: Can you please explain how the changes in this file relate to passing the background color to the renderer?

Comment (eulertour) [reply manim/mobject/types/image_mobject.py:96]: Those changes are actually part of a different change, but I thought submitting a new PR would be too slow. I'll add them to the description.

**Review by behackl — APPROVED** [review]
LGTM

**Review by naveen521kk — APPROVED** [review]
LGTM!

**Pull request merged**